software/contrib/master_clock.py

```diff
@@ -209,16 +182,19 @@ def average(self, list):
 
     '''Show running status'''
     def screen1(self):
```

Comment:
I don't think this method is called anymore, so it can be removed too.

Reply:
Agreed, good spot. Done.